includes/Modules/Analytics_4/Web_Tag.php

```diff
@@ -148,66 +141,40 @@ protected function enqueue_gtag_script() {
 		$gtag_opt = apply_filters( 'googlesitekit_gtag_opt', $gtag_opt );
 
 		if ( ! empty( $gtag_opt['linker'] ) ) {
-			$linker = wp_json_encode( $gtag_opt['linker'] );
-			$linker = sprintf( "gtag('set', 'linker', %s );", $linker );
-			wp_add_inline_script( 'google_gtagjs', $linker );
-		}
+			$gtag->add_command( 'set', array( 'linker', $gtag_opt['linker'] ) );
```

Comment:
This has the unexpected effect of changing the order in which the `linker` is set. In my testing, I saw it as the last command, where apparently the order can be significant.

See https://developers.google.com/tag-platform/devguides/cross-domain#set_up_cross-domain_linking

Let's update it to remain as the first command, when set.

Reply:
Thank you for pointing this out. I've addressed this by outputting all the commands (added via `add_command`) before the hardcoded `gtag("js")` command. Please let me know if that sounds good, thanks!

Reply:
Thanks, although that change would also print the `config` commands and maybe others before `js`. I think we want to keep the order as today for now, and we can see about building in some kind of priority sorting maybe in the future.

Reply:
Thank you!

The `config` commands do appear after `js` with the current ordering in place in this PR. See:

![image](https://github.com/google/site-kit-wp/assets/20284937/4e0d0e37-79a8-4896-b57a-106c316131c4)

This imitates the order that we have today. I absolutely agree, some kind of mechanism to sort these by priority and position would be great, I can open an issue to track that.

Do you think what we have in this PR is sufficient for now?

Reply:
My mistake, thanks for clarifying! 👍